MarsTestAutomation/StepDefinition/SellerProfileCertificationSteps.cs

```diff
@@ -0,0 +1,63 @@
+﻿using MarsTestAutomation.Pages;
+using MarsTestAutomation.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using TechTalk.SpecFlow;
+
+namespace MarsTestAutomation.StepDefinition 
+{
+    [Binding]
+    class SellerProfileCertificationSteps: CommonDriver
+    {
+        private ProfilePage CertificationObj;
+
+        [BeforeScenario]
+        public void Initialization()
+        {
+            CertificationObj = new ProfilePage();
```

Comment:
good use of creating objects

Reply:
i would also just rename the object to CertificationPage as its more clear

Reply:
done